test/Identity.js

```diff
@@ -0,0 +1,32 @@
+const { test } = require('./lib')
+
+const { of, empty, chainRec } = require('../src/quasi.js')
+
+const Identity = require('../examples/Identity.js')
+const List = require('../examples/List.js')
+
+test('eq', t => {
+  const a = 'a'
+  const as = List.fromArray([a])
+
+  t.eqFL(Identity(0), chainRec((next, done, v) => Identity(v === 0 ? done(v) : next(v - 1)), 10))
+  t.eqFL(Identity(10), chainRec((_, done, v) => Identity(done(v)), 10))
+  t.ok(of(a).equals(Identity.of(a)))
+  t.eqFL(Identity(of(a)), of(a).traverse((a) => Identity(a), Identity.of))
+  t.eqFL(of(Identity(of(a))), of(a).extend((a) => Identity(a)))
+  t.eqFL(of(Identity(of(empty))), empty.extend((a) => Identity(a)))
```

Comment:
`Identity` besides `a-> Identity a` is also `TypeRep Identity`  and I wanted to be more explicit that this is a function, but `Identity` would work as well.